src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -1016,18 +1023,25 @@ bool clang_cpp_convertert::name_param_and_continue(
    *     (done as part of the clang_c_converter's get_function_param flow)
    */
 
-  assert(id.empty() && name.empty());
```

Comment:
Pity that it's gone now. We should restore it. Helps documenting what this function expects.